### PR TITLE
NR.1: fix formatting of example

### PR DESCRIPTION
**Pull request by ewoudvc**
one space short to qualify as an example for MarkDown apparently

### CODE REVIEW

File: CppCoreGuidelines.md [hunk 1]
```diff
@@ -17997,23 +17997,23 @@ This leads to longer programs and more errors caused by uninitialized and wrongl
...
-           // ...
-           i = g(x, c);
-       }
-       return i;
+    int use(int x)
+    {
+        int i;
+        char c;
+        double d;
+
```
Comment (jwakely) [inline CppCoreGuidelines.md]: Could you remove the whitespace here, it causes the CI checks to fail.

File: CppCoreGuidelines.md [hunk 2]
```diff
@@ -17997,23 +17997,23 @@ This leads to longer programs and more errors caused by uninitialized and wrongl
...
-       }
-       return i;
+    int use(int x)
+    {
+        int i;
+        char c;
+        double d;
+ 
+        // ... some stuff ...
+
```
Comment (jwakely) [inline CppCoreGuidelines.md]: And here too.

Comment (ewoudvc) [inline CppCoreGuidelines.md]: ok, just pushed again

Comment (ewoudvc) [inline CppCoreGuidelines.md]: seems even more have failed. I'll revert my 2 commits, and try by just changing the first 2 lines of the example, and see what CI does with those

Comment (ewoudvc) [inline CppCoreGuidelines.md]: see https://github.com/isocpp/CppCoreGuidelines/pull/907

Comment (ewoudvc) [inline CppCoreGuidelines.md]: seem to have passed now on the other pull request

**Comment by ewoudvc** [comment]
filed a second pull request, with less spaces, that didn't break the build
see https://github.com/isocpp/CppCoreGuidelines/pull/907

**Pull request closed without merging**